src/repository/DeviceKeyFileRepository.js

```diff
@@ -13,31 +13,20 @@ class DeviceKeyFileRepository {
     this._fileManager = new FileManager(path);
```

Comment:
yeah, Ill add the interface. About `create`, we never use it anywhere, but I can return it